mypy.ini

```diff
@@ -1,4 +1,5 @@
 [mypy]
+disable_error_code =annotation-unchecked
```

Comment:
this subpresses all the `tests/geopandas/test_pydantic.py:20: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]` type warnings/notes in the mypy pre-commit stage